### ISSUE TITLE
Edit Virtual Shopping Cart (duplicate)

### ISSUE THREAD

**Issue by kowalczj** [kowalczj/shop24#12]
I also want to be able to edit or remove items from my shopping cart because I may change my mind in what I want to purchase.

**Comment by crisischris** [comment]
combine w/ other virtual shopping cart story

**Comment by OSSAVIVI** [comment]
Agreed, this should be combined with #11 Save Virtual Shopping Cart
